ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -904,3 +904,101 @@ def test_ipahealthcheck_trust_agent_member_check(self):
         for check in data:
             assert check["result"] == "SUCCESS"
             assert check["kw"]["key"] == self.master.hostname
+
+
+class TestIpaHealthCheckWithExternalSelfSignedCA(IntegrationTest):
+    """
+    Tests to run and check whether ipa-healthcheck tool reports correct status when
+    IPA Master has external self signed CA.
+    """
+
+    @classmethod
+    def install(cls, mh):
+        result = install_server_external_ca_step1(cls.master)
+        assert result.returncode == 0
+        root_ca_fname, ipa_ca_fname = tasks.sign_ca_and_transport(
+            cls.master, paths.ROOT_IPA_CSR, ROOT_CA, IPA_CA
+        )
+
+        result = install_server_external_ca_step2(
+            cls.master, ipa_ca_fname, root_ca_fname
+        )
+        assert result.returncode == 0
```

Comment:
You don't need this line. `raiseonerr=True` is set by default for the method.

Reply:
Removed the assert statement